src/main/java/core/model/WorldDetailsManager.java

```diff
@@ -7,10 +7,10 @@
 public class WorldDetailsManager {
 
 	private static WorldDetailsManager WMANAGER = null;
-	private WorldDetailLeague[] leagues;
-	private HashMap<Integer, WorldDetailLeague> countryMap = new HashMap<Integer, WorldDetailLeague>();
-	private HashMap<Integer, WorldDetailLeague> leagueMap = new HashMap<Integer, WorldDetailLeague>();
-	private int totalUsers;
+	private static WorldDetailLeague[] leagues;
+	private static HashMap<Integer, WorldDetailLeague> countryMap = new HashMap<Integer, WorldDetailLeague>();
+	private static HashMap<Integer, WorldDetailLeague> leagueMap = new HashMap<Integer, WorldDetailLeague>();
+	private static int totalUsers;
```

Comment:
Any reason for making all these `static`?  `WorldDetailsManager` seems to be a singleton, so `WorldDetailsManager.instance()` should do, and you can then call `getWorldDetailLeagueByCountryId` on it.

Reply:
no I guess we can go for your approach ... for my info what are the advantages of not making it static ?  I thought it would be cleaner this way ...

Reply:
I usually dislike static because they are “viral” (makes other members static in the same class), and it makes it hard to unit tests, but here this is more for consistency: “state” classes like `HOVerwaltung`, `HOLogger`, `HOParameter`, `DBManager`, `User`, etc. etc. are all singletons, so this one would be the odd one out.

Reply:
Ok I am convinced ... I will do that then :-)